tests/test_opening.py

```diff
@@ -137,6 +137,9 @@ def test_v2_open_sigs_restart(node_factory, bitcoind):
         l1.rpc.openchannel_signed(chan_id, psbt)
 
     l2.daemon.wait_for_log('Broadcasting funding tx')
+    l2.daemon.wait_for_log('sendrawtx exit 0')
+    txid = l2.rpc.listpeers(l1.info['id'])['peers'][0]['channels'][0]['funding_txid']
+    assert txid in bitcoind.rpc.getrawmempool()
     bitcoind.generate_block(6)
```

Comment:
generate_block can do this for you, BTW!

In particular, `generate_block(6, wait_for_mempool=l2.rpc.listpeers(l1.info['id'])['peers'][0]['channels'][0]['funding_txid'])`